test/__snapshots__/mathml-spec.js.snap

```diff
@@ -77,6 +77,46 @@ exports[`A MathML builder should make prime operators into <mo> nodes 1`] = `
 
 `;
 
+exports[`A MathML builder should render boldsymbol with the correct mathvariants 1`] = `
+
+<math>
+  <semantics>
+    <mrow>
+      <mrow>
+        <mi mathvariant="bold-italic">
+          A
+        </mi>
+        <mi mathvariant="bold-italic">
+          x
+        </mi>
+        <mn>
+          2
+        </mn>
+        <mi mathvariant="bold-italic">
+          k
+        </mi>
+        <mi mathvariant="bold-italic">
+          ω
+        </mi>
+        <mi mathvariant="bold-italic">
+          Ω
+        </mi>
+        <mi>
+          ı
+        </mi>
+        <mo>
+          +
+        </mo>
+      </mrow>
```

Comment:
I think everything in this row should have `mathvariant="bold-italic"` set.

Reply:
Are you sure? These include the imaginary number symbol, as well as the plus symbol, for which `mathvariant="bold"` is not set even for `\mathbf`.

Reply:
Rendering the following on quicklatex.com:
```
\boldsymbol{Ax2k\omega\Omega\imath+} \\
Ax2k\omega\Omega\imath+
```
results in:
<img width="303" alt="screen shot 2017-12-08 at 11 49 29 pm" src="https://user-images.githubusercontent.com/1044413/33792601-a99a1a50-dc72-11e7-89bd-943e3a07b288.png">

Reply:
The symbols for `\imath` and `+` that are present in the font looks as follows.

<img width="222" alt="screen shot 2017-12-09 at 10 33 00 am" src="https://user-images.githubusercontent.com/4722472/33798390-6044a4d8-dccc-11e7-99f9-5c2a6213560b.png">

This is from inspecting the element and manually adding `\boldsymbol` to the CSS. I don't know how to make KaTeX do that, and given the symbols don't match, should this be done at this stage?

Reply:
Update: the problem is that bold italic symbols for `\imath`, `+`, and `2` are contained in a different font: KaTeX-main. See screenshot below.

<img width="250" alt="screen shot 2017-12-09 at 11 28 27 am" src="https://user-images.githubusercontent.com/4722472/33798806-398d9a5e-dcd4-11e7-9c7e-7bbf9b3274a9.png">

Reply:
Turns out the same situation occurs for `\mathit` - using that as a template, I was able to implement `\imath` and `2`. `+` is still not correct, because for some reason the font doesn't get applied to it, will look at that shortly.